### PR TITLE
Alpha fixes

### PR DESCRIPTION
**Pull request by saifulbkhan**
Several fixes for v0.10.0-alpha. Each fix is an individual commit.

### CODE REVIEW

File: src/gui/mzroll/mzfileio.cpp [hunk 1]
```diff
@@ -1057,15 +1057,15 @@ void mzFileIO::_readPeakTablesFromSQLiteProject(const vector<mzSample*> newSampl
             group->setAdduct(DB.findAdductByName(group->getAdduct()->getName()));
 
         // assign group to bookmark table if none exists
-        if (group->searchTableName.empty())
-            group->searchTableName = "Bookmark Table";
+        if (group->tableName().empty())
+            group->setTableName("Bookmark Table");
```
Comment (shubhra-agrawal) [inline src/gui/mzroll/mzfileio.cpp]: Make sure that the table name is updated when the bookmarked group is merged to any peak table.

File: src/gui/mzroll/tabledockwidget.cpp [hunk 2]
```diff
@@ -660,7 +660,7 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
                               return group.getCompound()->type() == Compound::Type::PRM;
                             });
   bool prmGroupExists = prmGroupAt != end(allgroups);
-  bool includeSetNamesLines = true;
+  bool includeSetNamesLines = false; // when should this be true?
```
Comment (shubhra-agrawal) [inline src/gui/mzroll/tabledockwidget.cpp]: This should be true when the user selects the "Group Summary Matrix Format with Set Names" option.

Comment (shubhra-agrawal) [reply src/gui/mzroll/tabledockwidget.cpp]: Let me know if that option is working.

**Comment by shubhra-agrawal** [comment]
I'm not quite sure about the bug or fix in the persistent message box. The rest looks fine; I've added comments where I want you to double-check something.

File: src/gui/mzroll/tabledockwidget.cpp (hunk 2 above)
Comment (saifulbkhan) [reply src/gui/mzroll/tabledockwidget.cpp]: I see. Thanks for the info. I'll make sure it works.

File: src/gui/mzroll/mzfileio.cpp (hunk 1 above)
Comment (saifulbkhan) [reply src/gui/mzroll/mzfileio.cpp]: Will fix.

**Comment by saifulbkhan** [comment]
@shubhra-agrawal The message box that is supposed to inform the user about ongoing save process, before they exit El-MAVEN, was not staying for the entire duration of the save process. This was because it was stack allocated in a function (`MainWindow::saveProject`) and was destroyed as soon as the function returned. This made El-MAVEN window interact-able, during final save which is something that we do not want. I only made the message box heap allocated so that it lives until the application destroys itself (and therefore cleaning any associated allocations, including this message box).

**Pull request merged**